src/test/java/teammates/test/cases/pagedata/InstructorCourseStudentDetailsPageDataTest.java

```diff
@@ -131,8 +131,8 @@ private void createStudentProfile(String email, String pictureKey) {
     private InstructorCourseStudentDetailsPageData createData() {
         createCommonData();
 
-        return new InstructorCourseStudentDetailsPageData(AccountAttributes.builder().build(),
-                dummySessionToken, inputStudent, inputStudentProfile, hasSection);
+        return new InstructorCourseStudentDetailsPageData(AccountAttributes.builder().build(), dummySessionToken,
```

Comment:
Just curious why here we're initializing `AccountAttributes` with `googleId` as `null` instead of some hardcoded string or even using a data bundle for the whole thing ?

Reply:
If I am not wrong, the userID GET parameter in the URL is only used when the real user is masquerading as someone else. This page does not use this parameter because whether a real instructor or an admin disguising as an instructor, the page should look the same. hence, the field doesn't matter here and can just be `null`